docker/rootfs/usr/local/go2rtc/create_config.py

```diff
@@ -64,8 +64,14 @@
 # as source for frigate and the integration supports HLS playback
 if go2rtc_config.get("rtsp") is None:
     go2rtc_config["rtsp"] = {"default_query": "mp4"}
-elif go2rtc_config["rtsp"].get("default_query") is None:
-    go2rtc_config["rtsp"]["default_query"] = "mp4"
+else:
+    if go2rtc_config["rtsp"].get("default_query") is None:
+        go2rtc_config["rtsp"]["default_query"] = "mp4"
+    # perform environment variable substitution for the go2rtc rtsp creds
+    if go2rtc_config["rtsp"].get("username") is not None:
+        go2rtc_config["rtsp"]["username"] = go2rtc_config["rtsp"]["username"].format(**FRIGATE_ENV_VARS)
+    if go2rtc_config["rtsp"].get("password") is not None:
+        go2rtc_config["rtsp"]["password"] = go2rtc_config["rtsp"]["password"].format(**FRIGATE_ENV_VARS)
```

Comment:
```suggestion
    if go2rtc_config["rtsp"].get("default_query") is None:
        go2rtc_config["rtsp"]["default_query"] = "mp4"

    # perform environment variable substitution for the go2rtc rtsp creds

    if go2rtc_config["rtsp"].get("username") is not None:
        go2rtc_config["rtsp"]["username"] = go2rtc_config["rtsp"]["username"].format(**FRIGATE_ENV_VARS)

    if go2rtc_config["rtsp"].get("password") is not None:
        go2rtc_config["rtsp"]["password"] = go2rtc_config["rtsp"]["password"].format(**FRIGATE_ENV_VARS)
```

Reply:
This should be resolved now, added the two blank lines